samples/excel/30-events/events-chart-activated.yaml

```
@@ -28,7 +29,16 @@ script:
 
         async function chartActivated(event) {
             await Excel.run(async (context) => {
-                console.log("The pie chart is the active chart. ID: " + event.chartId);
+                // Retrieve the worksheet.
+                var sheet = context.workbook.worksheets.getActiveWorksheet();
```

The samples favor `const` and `let` over `var`.
```suggestion
                const sheet = context.workbook.worksheets.getActiveWorksheet();
```